packages/components/pie-input/test/component/pie-input.spec.ts

```diff
@@ -700,5 +753,27 @@ test.describe('PieInput - Component tests', () => {
             await page.click('button[type="reset"]');
             expect(await page.evaluate(() => document.querySelector('pie-input')?.value)).toBe('');
         });
+
+        test('should not submit the value for disabled inputs', async ({ page }) => {
+            // Arrange
+            await page.setContent(`
+                <form id="testForm" action="/foo" method="POST">
+                    <pie-input type="text" name="username" disabled></pie-input>
+                    <pie-input type="text" name="email"></pie-input>
+                    <button type="submit">Submit</button>
+                </form>
+                <div id="formDataJson""></div>
+            `);
+
+            await setupFormDataExtraction(page, '#testForm', '#formDataJson');
+
+            // Act
+            await page.click('button[type="submit"]');
+
+            const formDataObj = await getFormDataObject(page, '#formDataJson');
+
+            // Assert
+            expect(formDataObj).toStrictEqual({ email: '' });
```

Comment:
entirely up to you, but I wonder if it is worth adding a value to the disabled input to verify that does not submit either